homeassistant/components/device_tracker/icloud.py

```diff
@@ -47,24 +47,22 @@ def keep_alive(now):
         api.authenticate()
         _LOGGER.info("Authenticate against iCloud")
 
-    track_utc_time_change(hass, keep_alive, second=0)
+    track_utc_time_change(hass, keep_alive, second=0, minute=4)
```

Comment:
Its firing every 4 minutes at the moment!

should I do like line 81:
`minute=range(0, 60, config[CONF_INTERVAL]`

Or does it actually take `/4`?